loafwallet.xcodeproj/project.pbxproj

```diff
@@ -2068,7 +2064,8 @@
 				22A9A9AA1DF64B97000F0016 /* iphone.modulemap */,
 				22A9A9AB1DF64B97000F0016 /* iphonesim.modulemap */,
 			);
-			path = libbz2;
+			name = libbz2;
+			path = Modules/libbz2;
 			sourceTree = "<group>";
```

Comment:
Check this if you would @kcw-grunt , did your workspace have the wrong path for this folder?